src/r2mm/launching/runners/linux/SteamGameRunner_Linux.ts

```diff
@@ -68,8 +69,10 @@ export default class SteamGameRunner_Linux extends GameRunnerProvider {
 
         LoggerProvider.instance.Log(LogSeverity.INFO, `Steam directory is: ${steamDir}`);
 
+        const steamCmd = steamDir.indexOf(path.join(homedir(), '.var', 'app', 'com.valvesoftware.Steam')) == 0 ? `flatpak run --branch=stable --arch=x86_64 --command=/app/bin/steam-wrapper --file-forwarding --filesystem=home/.config/r2modmanPlus-local com.valvesoftware.Steam` : `"${steamDir}/steam.sh"`;
```

Comment:
I'm using Flatpak on immutable Kinoite. By default, r2modman seems to detect the steam directory under `/var/home/<username>/.var/app/com.valvesoftware.Steam/.local/share/Steam`. Since Kinoite `/home` is a symlink to `/var/home`, `homedir()` returns a path starting with `/home/<username>` and not `/var/home/<username>`. So while that path will be contained in steamDir it's index won't be 0.

 I changed the ` == 0` to ` >= 0` and was able to launch via flatpak successfully because of that.